src/Our.Umbraco.InnerContent/PropertyEditors/SimpleInnerContentPreValueEditor.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using Our.Umbraco.InnerContent.Helpers;
+using Umbraco.Core;
+using Umbraco.Core.Models;
+using Umbraco.Core.PropertyEditors;
+
+namespace Our.Umbraco.InnerContent.PropertyEditors
+{
+    public class SimpleInnerContentPreValueEditor : InnerContentPreValueEditor
+    {
+        [PreValueField("contentTypes", "Content Types", "~/App_Plugins/InnerContent/views/innercontent.doctypepicker.html", Description = "Select the content types to use as the data blueprint.")]
+        public string[] ContentTypes { get; set; }
```

Comment:
@mattbrailsford Are we sure that we want to have these specific fields for the "Simple" type?
I think we said yes, but wanted to make sure.

Reply:
I guess really we just want the contenttypes property, all the others are subjective I guess

Reply:
Urgh, it doesn't work. The way the properties are read in by Umbraco means that the current class fields are displayed first, then the inherited class fields are displayed next - so the "Content Types" appears at the bottom. 😕 
There isn't an "Order" field on the `PreValueField` attribute.

Reply:
I've ended up re-including `ContentTypes` property in the StackedContent class, (using the `new` keyword to hide the inherited property), it's not ideal, but I don't want it to be a showstopper just yet.

Reply:
Had a quick look at Umbraco's `PreValueEditor` base class, we might be able to manipulate the ordering of the fields, there's a `Fields` property we can access: https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Core/PropertyEditors/PreValueEditor.cs#L99
It feels a bit hacky 😈

Reply:
Doesn't look all that hacky given the attribute approach is just a vanity layer on top of the fields collection.

Reply:
I'll give it a try!